agent/checks/docker.go

```diff
@@ -151,11 +158,17 @@ func (c *DockerClient) CreateExec(containerID string, cmd []string) (string, err
 }
 
 func (c *DockerClient) StartExec(containerID, execID string) (*circbuf.Buffer, error) {
-	data := struct{ Detach, Tty bool }{Detach: false, Tty: true}
+	data := struct{ Detach, Tty bool }{Detach: false, Tty: false}
```

Comment:
why did tty change?

Reply:
never mind i just saw the commit message

Reply:
The main reason was that the older client was using `Tty: false`. I *think* pseudo-terminals are required for interactive sessions but I'll double check.

Reply:
Looks like my assumptions was correct. Will merge.